common/consistenthash/consistenthash.go

```diff
@@ -22,6 +22,17 @@ func (r *Ring) search(key string) int {
     // YOUR CODE GOES HERE //
     /////////////////////////
 
+    r.Lock()
+    defer r.Unlock()
+
+    strHash := hashId(key)
+
+    for i, node := range r.Nodes {
+        if node.HashId > strHash {
+            return i
+        }
+    }
```

Comment:
It's actually right, I just misread the code and though that it would return the last i value. Lab is fine.